### PR TITLE
updating branch

### PR DESCRIPTION
**Pull request by Aqui1983**

### CODE REVIEW

**Review by Aqui1983 — COMMENTED** [review]
everything is fine

**Review by scoopcash — APPROVED** [review]

**Comment by Aqui1983** [comment]
thanks

**Review by Aqui1983 — COMMENTED** [review]
everything is fine

**Review by scoopcash — COMMENTED** [review]
wow

**Review by marcleipold — APPROVED** [review]

File: jerami/jerami.ipynb [hunk 1]
```diff
@@ -44,38 +37,121 @@
...
+   "execution_count": 521,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import datetime as dt\n",
-    "import pytz\n",
-    "#today = dt.date.today()\n",
-    "# symbols we will be looking at\n",
-    "btc = \"BTCUSD\"\n",
-    "#spy = \"SPY\"\n",
```
Comment (Aqui1983) [inline jerami/jerami.ipynb:61]: ok

**Pull request merged**